tests/components/systemmonitor/test_psi_sensor.py

```diff
@@ -0,0 +1,92 @@
+"""Test System Monitor PSI sensor."""
+
+from unittest.mock import Mock, patch
+
+import pytest
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.components.systemmonitor.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from tests.common import MockConfigEntry
+
+MOCK_PRESSURE_INFO = {
```

Comment:
Add mock info in `conftest.py` to the existing mock and these sensors should be already covered by the existing tests.